pom.xml

```diff
@@ -587,7 +587,6 @@ under the License.
                 <groupId>org.apache.accumulo</groupId>
                 <artifactId>accumulo-minicluster</artifactId>
                 <version>${accumulo.version}</version>
-                <scope>test</scope>
```

Comment:
we shouldn't have support for mini/mock instances outside of tests and examples.  The way I probably would have suggested avoiding the dependency would be to have the daos/connections to the underlying persistence passed in rather than instantiated.  i would create a task in jira to address this.